src/transaction/types.ts

```diff
@@ -6,6 +6,8 @@ export enum TxBodyKeys {
   CERTIFICATES = 4,
   WITHDRAWALS = 5,
   META_DATA_HASH = 7,
+  VALIDITY_INTERVAL_START = 8,
+  // MINT = 9, // unsupported in current version
```

Comment:
One idea, I would add mint field here so if someone passes a Mary tx with mint field the check passes, and I would add a special condition to the parsing funtion so if a mint field is present we throw a specific error. In this we would just throw invalid tx or something and I expect somebody to be totally confused why we dont allow mary txs.

Reply:
Fixed.